svm-decode/src/lib.rs

```diff
@@ -83,23 +83,79 @@ pub fn svm_decode_instructions(
     batch: &RecordBatch,
     allow_decode_fail: bool,
 ) -> Result<RecordBatch> {
-    let data_col = batch.column_by_name("data").unwrap();
+    let data_col = batch
+        .column_by_name("data")
+        .context("data column not found in instructions batch")
+        .unwrap();
     let data_array = data_col.as_any().downcast_ref::<BinaryArray>().unwrap();
 
-    let account_arrays: Vec<&BinaryArray> = (0..10)
-        .map(|i| {
-            let col_name = format!("a{}", i);
-            let col = batch.column_by_name(&col_name).unwrap();
-            col.as_any().downcast_ref::<BinaryArray>().unwrap()
-        })
-        .collect();
+    let mut account_arrays: Vec<Option<BinaryArray>> = Vec::with_capacity(10);
+
+    for i in 0..10 {
+        let col_name = format!("a{}", i);
+        if let Some(col) = batch.column_by_name(&col_name) {
+            if let Some(binary_array) = col.as_any().downcast_ref::<BinaryArray>() {
+                account_arrays.push(Some(binary_array.clone()));
+            } else {
+                account_arrays.push(None);
+            }
+        }
+    }
+
+    if signature.accounts_names.len() > 10 {
```

Comment:
Now handling the case where the user asked to decode more than 10 accounts. Previous versions had this simplified and would not decode them.